src/mon/Paxos.cc

```diff
@@ -266,12 +266,12 @@ void Paxos::handle_collect(MonOpRequestRef op)
     logger->inc(l_paxos_collect);
     logger->inc(l_paxos_collect_keys, t->get_keys());
     logger->inc(l_paxos_collect_bytes, t->get_bytes());
-    utime_t start = ceph_clock_now();
 
+    auto start = ceph::coarse_mono_clock::now();
     get_store()->apply_transaction(t);
+    auto end = ceph::coarse_mono_clock::now();
 
-    utime_t end = ceph_clock_now();
-    logger->tinc(l_paxos_collect_latency, end - start);
+    logger->tinc(l_paxos_collect_latency, totimespan(end - start));
   } else {
```

Comment:
We could also add another overload to logger->tinc that can take signedspan. That would be my preference in this case.

Reply:
I didn't think it would make sense to add a negative timespan to the logger so I kinda prefer forcing the caller to explicitly convert to the unsigned version (and eat an assertion early via `totimespan` if they messed up).